### DISCUSSION TITLE
'xarray_extras.csv.to_csv' Doesn't Work

### DISCUSSION THREAD

**Discussion by AvitayGT** [pydata/xarray#5141] [Q&A] [answered]
Hi guys,

I am trying to use the function in the title, from the xarray_extras package, documented here at page #9, under "3.3 csv" : **[LINK](https://xarray-extras.readthedocs.io/_/downloads/en/v0.3.0/pdf/)** 

But when I do, I surprisingly receive the following error:

![image](https://user-images.githubusercontent.com/81641993/114318556-3377b100-9b16-11eb-9caa-4cdee2e1affc.png)

Does anybody know what's the problem? 

- Avitay

**Comment by max-sixty** [comment, marked as answer]
It needs to be imported like so:

```
In [3]: from xarray_extras import csv

In [4]: csv
Out[4]: <module 'xarray_extras.csv' from '/Users/maximilian/.asdf/installs/python/3.8.8/lib/python3.8/site-packages/xarray_extras/csv.py'>

In [5]: csv.to_csv
Out[5]: <function xarray_extras.csv.to_csv(x: xarray.core.dataarray.DataArray, path: str, *, nogil: bool = True, **kwargs)>
```

**Reply by AvitayGT** [reply]
Thank you very much!!
